src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final Color DEFAULT_COLOR = Color.WHITE;
+    public static final int DEFAULT_COUNT = 10;
+    private int figureNumber;
+    private Random random = new Random();
+    
+    public Figure getRandomFigure() {
+        figureNumber = random.nextInt(FIGURE_COUNT);
+        ColorSupplier colorSupplier = new ColorSupplier();
+        switch (figureNumber) {
+            case 1:
+                return new Circle(colorSupplier.getRandomColor(),
+                        random.nextInt(FIGURE_COUNT));
```

Comment:
```suggestion
                return getRandomCircle();
```
lets create private methods to get random figure and invoke it in each case